contents/newsletter/product-pricing-advice.md

```diff
@@ -0,0 +1,175 @@
+
+For example, our early pricing was wildly different from our pricing now. It was a monthly subscription with a free trial and way fewer events.
+
+![Early pricing](https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_10_03_at_14_24_212x_097c5fc3f5.png)
+
+## 2. It’s OK to change your pricing
+
+What the above example shows is that your pricing is always going to be a bit wrong. We started as a monthly subscription but eventually changed to usage-based. We had paid self-hosted product but made the tough decision to [sunset it](/blog/sunsetting-helm-support-posthog).
+
+To reach your full potential, you will need to change your pricing. Your product evolves, so should your pricing. All of your favorite companies do it:
+
+- **Notion** used to be paid only until [2020 when they added a free tier](https://www.notion.com/releases/2020-05-19) for personal use. They’ve since [expanded the free tier to remove any paid personal plans](https://www.notion.com/blog/notion-introduces-business-plan-and-expanded-free-plan), just focusing on paid teams plans.
+
+- **Figma** launched an [enterprise plan in 2022](https://www.figma.com/blog/design-needs-everyone-new-plans-for-companies-that-design-together/) with more admin tools, workspaces, and access controls. They also [raised prices](https://www.figma.com/blog/billing-experience-update-2025/) for Figma Design in 2025.
+
+- **Intercom** reinvigorated and [doubled its growth](https://lennysvault.com/insights/growth-scaling-tactics/fba11d05-3cf2-4591-a811-995a1fcb325c) by launching its AI agent, Fin, with outcomes-based pricing.
+
+![Fin pricing](https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_10_13_at_16_41_562x_fd6884de36.png)
+
+Changing your pricing enables you to better align the value you create with what you charge for it. For example, our [data pipelines product](/customer-data-infrastructure/destinations) used to be charged based on events ingested, but then [moved to rows exported and triggered events](/blog/data-pipeline-pricing) to better align what users actually used.
+
+The important part of doing this is making sure you communicate the change well. Our tips for doing this include:
+
+- Communicating early. Figma announced pricing changes in December 2024 that took effect March 2025.
+
+- Give people a warning and a discount (grace period) if you are going to raise their prices.
+
+- Understand who is going to pay more or less. 98% of our data pipeline customers paid less, but for the 2% who paid more, we made sure to communicate why we did this and help lower the impact of the change.
+
+- We try to grandfather existing customers’ pricing unless it is more expensive or we have a very good reason not to.
+
+## 3. How you price is who you are
```

Comment:
I'd be tempted to make this the first point. It feels fundamental.

Reply:
Moved to started, rewrote, and added more to it